kiva/constants.py

```diff
@@ -41,87 +55,209 @@
 # C version.
 # --------------------------------------------------------------------
 
-FILL = 1
-EOF_FILL = 2
-STROKE = 4
-FILL_STROKE = 5
-EOF_FILL_STROKE = 6
+class DrawMode(IntFlag):
+    """Drawing mode flags.
+
+    FILL
+        Paint the path using the nonzero winding rule
+        to determine the regions for painting.
+
+    EOF_FILL
+        Paint the path using the even-odd fill rule.
+
+    STROKE
+        Draw the outline of the path with the
+        current width, end caps, etc settings.
+
+    Note that FILL and EOF_FILL are mutually exclusive, so modes 3
+    and 7 aren't valid.
+
+    Outlines are stroked after any filling is performed.
+    """
+    FILL = 1
+    EOF_FILL = 2
+    STROKE = 4
+    FILL_STROKE = 5
+    EOF_FILL_STROKE = 6
+
+FILL = DrawMode.FILL
+EOF_FILL = DrawMode.EOF_FILL
+STROKE = DrawMode.STROKE
+FILL_STROKE = DrawMode.FILL_STROKE
+EOF_FILL_STROKE = DrawMode.EOF_FILL_STROKE
 
 # -----------------------------------------------------------------------------
 # Font Constants
 # -----------------------------------------------------------------------------
 
-NORMAL = 0
-BOLD = 1
-ITALIC = 2
-BOLD_ITALIC = 3
+class FontStyle(IntFlag):
+    """Basic font styles."""
+    NORMAL = 0
+    BOLD = 1
+    ITALIC = 2
```

Comment:
You're using this for bit masks in places so it might be a good idea to call that out in the docstring somehow.